package/src/components/ChannelPreview/hooks/__tests__/useLatestMessagePreview.test.tsx

```diff
@@ -0,0 +1,317 @@
+            { id: 'Max', name: 'Max' },
+            { id: 'Ada', name: 'Ada' },
+            { id: 'Enzo', name: 'Enzo' },
+          ],
+          text: 'Max',
+        },
+        previews: [
+          { bold: false, text: '' },
+          { bold: false, text: 'Max' },
+        ],
+        status: 0,
+      });
+    });
+  });
+
+  it('should return latest command', async () => {
+    const { result } = renderHook(
+      () => useLatestMessagePreview(CHANNEL_WITH_MESSAGES_COMMAND, FORCE_UPDATE, LATEST_MESSAGE),
+      { wrapper: ChatProvider },
+    );
+    await waitFor(() => {
+      expect(result.current).toEqual({
+        created_at: 'L',
+        messageObject: {
+          args: 'string',
+          attachments: [],
+          channel: { data: { name: 'okechukwu' }, state: { messages: [] } },
+          cid: 'stridodong',
+          command: 'giphy',
+          command_info: {
+            name: 'string',
+          },
+          created_at: new Date('2021-02-12T12:12:35.862Z'),
+          deleted_at: new Date('2021-02-12T12:12:35.862Z'),
+          id: 'jbkjb',
+          user: {
+            id: 'okechukwu',
+          },
+        },
+        previews: [
+          { bold: true, text: '@okechukwu: ' },
+          { bold: false, text: '/giphy' },
+        ],
+        status: 0,
+      });
+    });
+  });
+
+  it('should return latest attachment', async () => {
+    const { result } = renderHook(
+      () =>
+        useLatestMessagePreview(CHANNEL_WITH_MESSAGES_ATTACHMENTS, FORCE_UPDATE, LATEST_MESSAGE),
+      { wrapper: ChatProvider },
+    );
+    await waitFor(() => {
+      expect(result.current).toEqual({
+        created_at: 'L',
+        messageObject: {
+          args: 'string',
+          attachments: [
+            {
+              actions: [],
+              asset_url: 'string',
+              author_icon: 'string',
+              author_link: 'string',
+              author_name: 'string',
+              color: 'string',
+              fallback: 'string',
+              fields: [],
+              file_size: 25,
+              footer: 'string',
+              footer_icon: 'string',
+              image_url: 'string',
+              mime_type: 'string',
+              og_scrape_url: 'string',
+              original_height: 5,
+              original_width: 4,
+              pretext: 'string',
+              text: 'string',
+              thumb_url: 'string',
+              title: 'string',
+              title_link: 'string',
+              type: 'string',
+            },
+          ],
+          channel: { data: { name: 'okechukwu' }, state: { messages: [] } },
+          created_at: new Date('2021-02-12T12:12:35.862Z'),
+          deleted_at: new Date('2021-02-12T12:12:35.862Z'),
+          id: 'ljkblk',
+          user: {
+            id: 'okechukwu',
+          },
+        },
+        previews: [
+          { bold: true, text: '@okechukwu: ' },
+          { bold: false, text: '🏙 Attachment...' },
```

Comment:
That's what the user sees so let's only assert that, what are your thoughts on that?